src/components/modals/SettingsModal/SettingsDialog.tsx

```diff
@@ -63,14 +64,44 @@ const getConfigFormFields = () => [
         type: "text",
     },
     {
-        helperText: "[JSON] Key to extract the log level from.",
+        helperText: (
+            <span>
+                [Structured] Key to extract the log level from. See
```

Comment:
```suggestion
                [Structured] Field containing each log event's level. See
```

Reply:
Should be relatively common, but we could be more explicit:

```suggestion
                [Structured] Key of the kv-pair containing each log event's level. See
```

Reply:
Sure